### PR TITLE
Adding shortcut to submit modal input on enter key

### PR DESCRIPTION
**Pull request by Parasrah**
Addressing issue #50

### CODE REVIEW

**Comment by Parasrah** [comment]
Seems it was already failing the tests, as there are two areas it is not following your linter. Also I'm not totally sure how to run it to verify that it is working as I expect, if you could point me in the right direction I would appreciate it

File: src/interface/modal.js [hunk 1]
```diff
@@ -29,10 +29,25 @@ Modal.prototype.open = function () {
...
+    if (e.keyCode === 13) {
+      done(e)
+    }
+  }
+
   function cancel () {
     self.emit('cancel')
   }
 
+  var ip = Array.prototype.slice.call(self.el.querySelectorAll('modal-input'))
```
Comment (t-mullen) [inline src/interface/modal.js]: The selector needs to be `.modal-input`

**Review by t-mullen — CHANGES_REQUESTED** [review]
Looks good, thanks! Selector is missing a dot, but works other than that.

**Comment by Parasrah** [comment]
Lmao my bad, I'll fix that right away :p

**Review by t-mullen — APPROVED** [review]

**Pull request merged**